### PR TITLE
Replace finstyle in favor of chefstyle

### PR DESCRIPTION
**Pull request by afiune**
Currently we are having problems with ChefDK pipeline. This commit is fixing the problems by replacing `finstyle` in favor of `chefstyle`.

Signed-off-by: Salim Afiune <afiune@chef.io>

### CODE REVIEW

**Comment by mwrock** [comment]
I'm generally in favor of the change but its gonna require quite alot of changing things around to make chefstyle happy.

File: test-kitchen.gemspec [hunk 1]
```diff
@@ -59,5 +59,7 @@ Gem::Specification.new do |gem|
   # style and complexity libraries are tightly version pinned as newer releases
   # may introduce new and undesireable style choices which would be immediately
   # enforced in CI
-  gem.add_development_dependency "finstyle",  "1.5.0"
+  #
+  # Replacing finstyle in favor of chefstyle
+  gem.add_development_dependency "chefstyle",  "0.4.0"
```
Comment (thommay) [inline test-kitchen.gemspec]: The pattern usually is to just have chefstyle in the Gemfile, and float it on master.
https://github.com/chef/chef/blob/master/Gemfile#L77

**Comment by thommay** [comment]
`chefstyle -a` is your friend :)

**Comment by afiune** [comment]
Doing it now! 👍

**Comment by afiune** [comment]
```129 files inspected, 16 offenses detected```

Much better!!! I can deal with those ^^^

**Comment by afiune** [comment]
### Yup... That was easy...


cc/ @test-kitchen/maintainers

File: test-kitchen.gemspec [hunk 2]
```diff
@@ -59,5 +59,7 @@ Gem::Specification.new do |gem|
   # style and complexity libraries are tightly version pinned as newer releases
   # may introduce new and undesireable style choices which would be immediately
   # enforced in CI
-  gem.add_development_dependency "finstyle",  "1.5.0"
+  #
+  # Replacing finstyle in favor of chefstyle
+  gem.add_development_dependency "chefstyle", "0.4.0"
```
Comment (schisamo) [inline test-kitchen.gemspec]: Do we need to peg a specific version here? We float on latest in our other Ruby projects like Chef, ChefDK, Omnibus etc.

**Pull request merged**